annotator/pom.xml

```diff
@@ -14,6 +14,11 @@
   </parent>
 
   <dependencies>
+    <dependency>
+      <groupId>org.genome_nexus.client</groupId>
+      <artifactId>genomeNexusPublicApiClient</artifactId>
+      <version>1.0.0</version>
```

Comment:
@onursumer do we need this? Seems like this has been replaced by jitpack artifacts no?

Reply:
@inodb good catch, this should be removed.